tutorials/optimize/opt_gemm.py

```diff
@@ -42,45 +43,51 @@
 import numpy
 import timeit
 
-# The size of the square matrix
-N = 1024
+# The size of the matrix
+N1 = 1024
+N2 = 1024
```

Comment:
I am thinking about making it more generic by having `M, N, K` here, so we have `[M, K] * [K, N] = [M, N]`. And we should make a note here saying that we are testing out on the square matrices only; the users should feel free to change the matrix shapes; and different shapes yield different results, sometimes TVM outperforms MKL.